src/server/server.py

```diff
@@ -32,8 +32,6 @@
 from src.util.ints import uint16
 from src.util.network import create_node_id
 
-log = logging.getLogger(__name__)
```

Comment:
In tests when there are two severs, the log name is the same. I wanted to pass different name so that I would know what server logged what. (Did the same thing for the fullnode I believe)